analytics/compfeeds_analytics/lending_borrowing/utils.py

```diff
@@ -0,0 +1,29 @@
+# We use Almgren & Chris (2000) algorithm for estimating execution cost.
+# Link: https://www.smallake.kr/wp-content/uploads/2016/03/optliq.pdf
+
+
+def estimate_execution_cost(volatility, notionalSize, price, TWAMM):
```

Comment:
I still don't follow this.  Why is TWAMM capitalized and not used?  Also, most of your variables are single letters and capitalized.  Could you use a consistent variable naming scheme and add type hints here too?